WireAPI/Sourcery/config.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+- ../Sources/WireAPI
+templates:
+- ./AutoMockable.stencil
```

Comment:
We could also unify the stencil over all targets (and just make unique ones if something differs)?

Reply:
Yeah that would be great.

Reply:
At least the `import`s differ.

Reply:
@caldrian  I don't understand what you mean about the imports? I can not spot hardcoded imports in the stencil files.

Reply:
Sorry, I thought you mean the stencils.